nextcloud_update.sh

```diff
@@ -377,15 +377,15 @@ we have removed Watchtower from this server. Updates will now happen for each co
     fi
     # Update selected images
     # Bitwarden RS
-    docker_update_specific 'bitwardenrs/server' "Bitwarden RS"
+    docker_update_specific 'bitwarden_rs' "Bitwarden RS"
     # Collabora CODE
-    docker_update_specific 'collabora/code' 'Collabora'
+    docker_update_specific 'code' 'Collabora'
     # OnlyOffice
-    docker_update_specific 'onlyoffice/documentserver' 'OnlyOffice'
+    docker_update_specific 'onlyoffice' 'OnlyOffice'
     # Full Text Search
-    docker_update_specific 'ark74/nc_fts' 'Full Text Search'
+    docker_update_specific 'fts_esror' 'Full Text Search'
     # Plex
-    docker_update_specific 'plexinc/pms-docker' "Plex Media Server"
+    docker_update_specific 'plex' "Plex Media Server"
```

Comment:
Instead of doing it hard, it could be made easy. Rename `plex` here to `plexinc/pms-docker` instead - if that's the name. 

I'd rather be precise than to get a possible future mixup. The current way works as expected.

Reply:
I would say this doesnt work because '/' isn't allowed as docker name: https://github.com/moby/moby/blob/be97c66708c24727836a22247319ff2943d91a03/daemon/names/names.go#L6

Why not use the container names to update the containers? IMO, this is the cleaner way to do this (and also recommended in the docs).

Reply:
OK, in any case, it needs testing, and as usual I don't have time. :(

1 month backlog right now, it's crazy.

Reply:
I did some tests. hopefully they are enough to convince you? :)
Sry for ramping up your workload! :/